model.js

```diff
@@ -32,18 +32,22 @@ exports.fetchApiArticles = () => {
       GROUP BY articles.article_id
       ORDER BY articles.created_at DESC;
       `).then(({ rows }) => {
-        return rows
+        return rows 
       })
 };
 
-/* Responds with:
+exports.fetchApiArticleComments = (article_id) => {
+    return db.query(`
+        SELECT *
+        FROM comments
+        WHERE article_id = $1
+        ORDER BY created_at DESC`,
+    [article_id])
+    .then(({ rows }) => {
+        if(rows.length === 0) {
```

Comment:
think about when you read an article on bbc news - if the article has no comments, you dont want to see an error just because nobody has commented on it right? An error will prevent us from loading the page and reading the article - in which case no new article would ever receive a comment because we'd never be able to read it in the first place! So you want to say the request for the comments was successful - you just didnt get any comments back - so we'll need a 200 test for no comments returned!
In which case your 404 test for this endpoint will fail - how could we fix that?